npm/podController.go

```diff
@@ -132,14 +145,14 @@ func (c *podController) needSync(eventType string, obj interface{}) (string, boo
 		return key, needSync
```

Comment:
About metrics.SEndErrorLogAndMetric(). I think we do not need send this information since it can be ignorable. If want to keep it, need to change "util.NpmID" -> util.PodID"

Reply:
good catch changed this in 3 locations.